docs/en/plan-deployment/cost.md

```diff
@@ -42,20 +42,35 @@ Key assumptions include:
-|           |   43  |  122  |     MSK (m5.large * 2, connector MCU * 1)              |      417       |  3   |     585
-|           |   43  |  122  |     None              |             |  3    |     168
-|1000RPS           |   396  |   122 |     Kinesis(On-demand)              |      576       |  14   |    1108 |
-|                         |  396   |  122   | Kinesis (Provisioned 10 shard)   |    146         |   14  |     678  |
-|           |  396   | 122   |     MSK (m5.large * 2, connector MCU * 2~3)              |      530       |  14  |     1062
-|           |  396   | 122   |     None              |            |  14   |     532
+| Request Per Second | ALB | EC2  |  Buffer type      | Buffer cost | S3   |  Total (USD/Month) |
+| ------------------ | --- | ---  |  --------------   | ----------- | ---  |  --------- |
+| 10RPS              |  7  |  122 |  Kinesis (On-Demand) |    36       |   3  |     168  |
+|                    |  7  |  122 |  Kinesis (Provisioned 2 shard)   |      22       |  3   |   154  |
+|                    |  7  |  122 |  MSK (m5.large * 2, connector MCU * 1)   |       417      |   3  |     549   |
+|                         | 7    |  122 |  None              |             |  3    |      132   |
+|100RPS           |   43  |  122  |  Kinesis(On-demand)              |      86       |  3   |     254 |
+|                         | 43    |   122 |  Kinesis (Provisioned 2 shard)   |      26       | 3    |     194  |
+|           |   43  |  122  |   MSK (m5.large * 2, connector MCU * 1)              |      417       |  3   |     585
+|           |   43  |  122 |      None              |             |  3    |     168
+|1000RPS           |   396  |   122 |      Kinesis(On-demand)              |      576       |  14   |    1108 |
+|                         |  396   |  122  |  Kinesis (Provisioned 10 shard)   |    146         |   14  |     678  |
+|           |  396   | 122  |      MSK (m5.large * 2, connector MCU * 2~3)              |      530       |  14  |     1062
+|           |  396   | 122   |      None              |            |  14   |     532
+
+## Data transfer
+When data is transferred from EC2 to the downstream data sink, there are associated costs. Below is the example of data transfer costs based on 1000 RPS and 1KB payload of a request.
+
+1. EC2 Network In: This does not incur any costs.
+2. EC2 Network Out: There are three data sink options:
+    - S3: Recommended to use VPC endpoint, which does not incur any costs.
+    - MSK: There will be costs for "$0.010 per GB in/out/between EC2 AZs", about $105/month.
+    - KDS: There are two ways to sink data from EC2 to KDS, through NAT or VPC endpoint.
+        - NAT: If there is one NAT, the toal cost is about $633/month, the cost consists of the following items:
+            1. "$0.045 per NAT Gateway Hour", $32/month.
+            2. "$0.045 per GB Data Processed by NAT Gateways", $601/month.
+        - VPC Endpoint: $147/month.
```

Comment:
The same. The cost consists of `per hour for VPC endpoint` and the `data processed by vpc endpoint`.

Reply:
Refactored it.